eradiate/scenes/atmosphere/_particles.py

```diff
@@ -370,18 +371,21 @@ def eval_sigma_t(self, spectral_ctx: SpectralContext) -> pint.Quantity:
     def eval_sigma_t_mono(self, w: pint.Quantity) -> pint.Quantity:
 
         ds = xr.open_dataset(self.dataset)
-        wavelengths = w.m_as(ds.w.attrs["units"])
-        interpolated_sigma_t = ds.sigma_t.interp(w=wavelengths)
-        sigma_t = to_quantity(interpolated_sigma_t)
+        ds_w_units = ds.w.attrs["units"]
```

Comment:
How about pre-computing those:
```suggestion
        ds_w_units = ureg.Unit(ds.w.attrs["units"])
```